server/plugin/webhook.go

```diff
@@ -1030,7 +1044,29 @@ func (p *Plugin) handleCommentAssigneeNotification(event *github.IssueCommentEve
 		return
 	}
 
+	mentionedUsernames := parseGitHubUsernamesFromText(event.GetComment().GetBody())
+
 	for _, assignee := range assignees {
+		usernameMentioned := false
+		for _, username := range mentionedUsernames {
+			if username == *assignee.Login {
+				usernameMentioned = true
+				break
+			}
+		}
+
+		if usernameMentioned {
+			switch splitURL[len(splitURL)-2] {
```

Comment:
have we already checked for its length?

Reply:
we have to check for this special case whether this is a pull request or an issue.

Reply:
I was talking about the length if it's smaller than 2. But it's already being checked.